pegasus/sites.v3/code.org/public/educate/it.md

```diff
@@ -27,70 +27,71 @@ Starting in July 2024, we will update the minimum versions required for browsers
           Browser
```

Comment:
opinion: I think it would be a little easier to parse this table if this header said `Browser July '23` so that it is clearer that the '24 column preempts the '23 column. here and for `Platform` below

Reply:
What I'm doing here is changing to a single pair of tables labeled "Browser" and "Platform"

When we modify this page to re-add the requirements for next school year, we can revisit the headers for clarity. I'm owning that work, so I'll keep it in mind.